### PR TITLE
Sf autoencoder

### PR DESCRIPTION
**Pull request by lucidtronix**

### CODE REVIEW

File: api_tutorials/autoencoder.py [hunk 1]
```diff
@@ -0,0 +1,329 @@
...
+from keras.models import Model
+import matplotlib.pyplot as plt
+from keras.layers import Input, Dense
+from keras.callbacks import TensorBoard
+from keras.datasets import mnist, cifar10
+from keras.layers import Input, Dense, Conv2D, MaxPooling2D, UpSampling2D, LSTM, RepeatVector
+
+
+def run():
+	#autoencode_faces()
```
Comment (lucidtronix) [inline api_tutorials/autoencoder.py]: use argparse here

File: api_tutorials/autoencoder.py [hunk 2]
```diff
@@ -0,0 +1,329 @@
...
+from Bio import Seq, SeqIO
+from keras import regularizers
+from keras import backend as K
+from keras.models import Model
+import matplotlib.pyplot as plt
+from keras.layers import Input, Dense
+from keras.callbacks import TensorBoard
+from keras.datasets import mnist, cifar10
+from keras.layers import Input, Dense, Conv2D, MaxPooling2D, UpSampling2D, LSTM, RepeatVector
+
```
Comment (lucidtronix) [inline api_tutorials/autoencoder.py:29]: Add comment assumes TensorFlow Dim ordering!
